src/components/ui/Chat/ChatBubble/index.tsx

```diff
@@ -12,10 +12,12 @@ export interface ChatBubbleProps extends Omit<HTMLAttributes<HTMLDivElement>, 'c
   variant: MessageVariant;
   /** The name of the user that sent the message. */
   senderName: string;
-  /** The text content of the message. */
-  content: string;
+  /** The timestamp of the message being sent. */
+  timestamp?: string;
   /** Determines if the name should be shown or not. */
   showName?: boolean;
+  /** Determines if timestamp should be shown or not. */
+  showTimestamp?: boolean;
```

Comment:
I don't think we need this prop. If you don't want to show the timestamp, you can just not pass it.

Reply:
But we want to hide the timestamp and mouseover the chat bubble will show it

Reply:
In that use case, we could still pass `null` or `undefined` to hide the name or timestamp.